Observations/obs.md

```diff
@@ -0,0 +1,152 @@
+
+Examples of observations include Serum Creatinine of 0.9mg/dL or Review of cardiopulmonary system is normal.
+ 
+## Available operations. 
+
+1. [List observations](#list-observations)
+2. [Create an observation](#create-an-observation)
+3. [Update an observation](#update-an-observation)
+4. [Delete an observation](#delete-an-observation)
+
+### List observations
+
+* #### List all observations.
+    
+    Quickly filter observations with given query parameters. 
+If not authenticated or authenticated user does not have sufficient privileges, a `401 Unauthorized` 
+status is returned.
+
+    #### Query Parameters
+
+    Parameter | Type | Description
+    --- | --- | ---
+    *patient* | `target_patient_uuid` | patient resource UUID
+    *concept* | `target_concept_uuid`| concept resource UUID (this parameter to be used with patient)
+
+    ```console
+    GET /obs?patient=070f0120-0283-4858-885d-a20d967729cf"
+     ```
+    
+* #### Query observations by UUID.
+
+    Retrieve an observation by its UUID.
+If not authenticated or authenticated user does not have sufficient privileges, a `401 Unauthorized` 
+status is returned.
+    
+    ```console
+    GET /obs/:target_observation_uuid
+    ```
+   
+### Create an observation
+
+* To Create an observation you need to specify below attributes in the request body. If not authenticated or authenticated user does not have sufficient privileges, a `401 Unauthorized` 
+status is returned.
+
+    #### Attributes
+
+    Parameter | Type | Description
+    --- | --- | ---
+    *person* | `Person UUID` | the Person this Obs is acting on.
+    *obsDateTime* | `String` | the time this Obs took place.
+    *concept* | `String` | the coded value/name given to an obs when it is made.
+    *location* | `String` | the location this Obs took place (was taken).
+    *order* | `String` | the order of an Obs.
+    *encounter* | `String` | what obs are collected and grouped together into. An encounter is a visit.
+    *accessionNumber* | `String` | a unique identifier assigned to each Obs.
+    *groupMembers* | `Array[]: Obs` |  a list of Obs grouped under this Obs
+    *comment* | `String` | the comment written in an Obs.
+    *value* | `String` | various values saved from a given Obs.
+    *status* | ENUM | the status of the observation
+    *interpretation* | ENUM | the enumerations related to the current state of being of an Obs
+    *voided* | `Boolean` | whether the obs is voided or not
+    
+   
+    ```console
+    POST /obs
+    {
+      "person": {
+        "uuid": "070f0120-0283-4858-885d-a20d967729cf",
+        "display": "1001MH - John Smith",
+        "links": [
+          {
+            "rel": "self",
+            "uri": "http://demo.openmrs.org/openmrs/ws/rest/v1/patient/070f0120-0283-4858-885d-a20d967729cf"
+          }
+        ]
+      },
+      "obsDatetime": "2016-11-10T07:37:31.000+0000",
+      "voided": false,
+      "status": "PRELIMINARY",
+      "interpretation": "NORMAL"
+    }
```

Comment:
Did you get this to work? I don't believe this is a valid observation. It doesn't include the concept (the question) or the value (the answer) for the observation.

How about this?

```
POST /obs
{
  "person": "070f0120-0283-4858-885d-a20d967729cf",
  "concept": "5089AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
  "obsDatetime": "2019-11-14T07:37:31.000+0000",
  "value": 70
}
```

Using [httpie](https://httpie.org/), I was able to use this to create an observation for John Smith (1001MH) on demo.openmrs.org:

```
http -a admin:Admin123 https://demo.openmrs.org/openmrs/ws/rest/v1/obs \
  person=070f0120-0283-4858-885d-a20d967729cf \
  concept=5089AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA \
  obsDatetime="2019-11-12T07:37:31.000+0000" \
  value=70
```